Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -111,6 +112,16 @@ class FakespotViewController: UIViewController, Themeable {
             turnOffButtonTitleA11yId: AccessibilityIdentifiers.Shopping.SettingsCard.turnOffButton,
             recommendedProductsSwitchA11yId: AccessibilityIdentifiers.Shopping.SettingsCard.recommendedProductsSwitch)
         settingsCardView.configure(settingsCardViewModel)
+
+        let noAnalysisCardViewModel = NoAnalysisCardViewModel(
+            cardA11yId: AccessibilityIdentifiers.Shopping.NoAnalysisCard.card,
+            headlineLabelText: .Shopping.NoAnalysisCardHeadlineLabelTitle,
+            headlineLabelA11yId: AccessibilityIdentifiers.Shopping.NoAnalysisCard.headlineTitle,
+            bodyLabelText: .Shopping.NoAnalysisCardBodyLabelTitle,
+            bodyLabelA11yId: AccessibilityIdentifiers.Shopping.NoAnalysisCard.bodyTitle,
+            footerLabelText: .Shopping.NoAnalysisCardFooterLabelTitle,
+            footerLabelA11yId: AccessibilityIdentifiers.Shopping.NoAnalysisCard.footerTitle)
```

Comment:
Do we intend to use this NoAnalysisCard in multiple places? Just wondering if it's possible to have those identifiers as defaults in that view model.